src/test/java/org/assertj/scripts/Convert_Junit_Assertions_To_Assertj_Test.java

```diff
@@ -0,0 +1,264 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2020 the original author or authors.
+ */
+package org.assertj.scripts;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+import static java.lang.String.format;
+
+/**
+ * Tests for convert script {@code src/main/scripts/convert-junit-assertions-to-assertj.sh}.
+ *
+ * @author XiaoMingZHM, Eveneko
+ */
+
+public class Convert_Junit_Assertions_To_Assertj_Test {
```

Comment:
Suggestion: make the tests parameterized as they all follow the same template.
Ex:
```
  @ParameterizedTest(name = "{0} should be converted to {1}")
  @CsvSource(value = {
      "assertEquals(0, myList.size());\n, assertThat(myList).isEmpty();\n",
      "foo.bar, foo.bar",
  })
  public void should_match_fields(String input, String expected) {
    tester.Start_Test(input, expected);
  }
```
If `CsvSource` is not clear enough, another option is `@MethodSource`:
```
  @ParameterizedTest(name = "{0} should be converted to {1}")
  @MethodSource("replace_assertEquals_checking_size_source")
  public void should_replace_assertEquals_checking_size() {
    tester.Start_Test(input, expected);
  }
```
where
```
Stream<Object> should_replace_assertEquals_checking_size_source() {
    return Stream.of(arguments("assertEquals(0, myList.size());\n", 
                               "assertThat(myList).isEmpty();\n"),
                     arguments("assertEquals(0, myList.size());\n", 
                               "assertThat(myList).isEmpty();\n"));
  }
```

Reply:
Use `@MethodSource` to refactor the code.
BUT! The performance of these tests are so slow because of the fileIO. How can we improve it? Less fileIO times?